src/lib/logic-function.js

```diff
@@ -47,4 +113,30 @@ function createAPI() {
 	});
 }
 
+function createAPIErrorResult({ error: e, message, json }){
+	const error = new VError(formatAPIErrorMessage(e), message);
+	error.asJSON = json;
+	return error;
+}
+
+function formatAPIErrorMessage(error){
+	error = normalizedApiError(error);
+
+	if (error.body){
+		if (typeof error.body.error === 'string'){
+			error.message = error.body.error;
+		} else if (Array.isArray(error.body.errors)){
+			if (error.body.errors.length === 1){
+				error.message = error.body.errors[0];
+			}
+		}
+	}
+
+	if (error.message.includes('That belongs to someone else.')){
+		error.canRequestTransfer = true;
+	}
+
+	return error;
+}
```

Comment:
This should be in `api.js` file maybe? I saw this repetitive code in every `cmd` file but it's more related with how we treat api errors.